core/src/main/java/org/apache/brooklyn/util/core/internal/ssh/sshj/SshjTool.java

```diff
@@ -273,13 +273,8 @@ private int copyToServer(Map<String,?> props, Supplier<InputStream> contentsSupp
 
     @Override
     public int copyFromServer(Map<String,?> props, String pathAndFileOnRemoteServer, File localFile) {
-        InputStream contents = acquire(new GetFileAction(pathAndFileOnRemoteServer));
-        try {
-            FileUtil.copyTo(contents, localFile);
-            return 0; // TODO Can we assume put will have thrown exception if failed? Rather than exit code != 0?
-        } finally {
-            Streams.closeQuietly(contents);
-        }
+        LocalDestFile localDestFile = acquire(new GetFileAction(pathAndFileOnRemoteServer));
```

Comment:
ops, you are right!
seems like the only test capturing that is in `SshToolAbstractIntegrationTest` which is marked as `integration` -- I'll try to fix it